src/maps/artifact.ts

```diff
@@ -0,0 +1,17 @@
+import { MapFunction } from '@/services/Mapper'
+import { ArtifactResponse } from '@/models/api/ArtifactResponse'
+import { Artifact } from '@/models/Artifact'
+
+export const mapArtifactResponseToArtifact: MapFunction<ArtifactResponse, Artifact> = function (source) {
+  return new Artifact({
+    id: source.id,
+    created: new Date(source.created),
+    updated: new Date(source.updated),
```

Comment:
There's a mapper for converting iso date strings to date objects. I think it's basically what you have here but we should use it still.

Reply:
Ah good call, updated that. Also revealed that I had a type issue with the `ArtifactResponse` since that was supposed to be a string and not a date... +1 for type safe mappers